models/playerFantasyModel.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (sequelize, type) => {
+    return sequelize.define('playerFantasyModel', {
+      id: {
+        type: type.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      points: {
+        type: type.INTEGER,
+        defaultValue: 0,
+      },
+      money: {
+        type: type.INTEGER,
+        defaultValue: 0,
+      }
+    })
+}
```

Comment:
```suggestion
module.exports = (sequelize, type) => {
    return sequelize.define('playerFantasyModel', {
      id: {
        type: type.INTEGER,
        primaryKey: true,
        autoIncrement: true
      },
      points: {
        type: type.INTEGER,
        allowNull: false,
        defaultValue: 0,
      },
      money: {
        type: type.INTEGER,
        allowNull: false,
        defaultValue: 0,
      }
    })
}
```
Aunque coloquen el defaultValue como 0, todavía colocará este valor a NULL si lo permiten. Es mejor que directamente digan que no se le puede colocar NULL, o lo validen de alguna manera